### PR TITLE
BioSchemas/Schema.org markup update

### PR DESCRIPTION
**Pull request by ricardoaat**
- Updated schema to accommodate Bioschemas [MolecularEntity version 0.4-draft](https://bioschemas.org/specifications/drafts/MolecularEntity/)
- Added IUPAC property

### CODE REVIEW

File: src/glados/schema_tags_generator.py [hunk 1]
```diff
@@ -173,16 +173,22 @@ def get_schema_obj_for_compound(chembl_id, request):
...
         metadata_obj['name'] = item['pref_name']
 
+    iupac_name = ""
     alternate_names_set = set()
     raw_synonyms = item['molecule_synonyms']
     if raw_synonyms is not None:
         for raw_syn in raw_synonyms:
             alternate_names_set.add(raw_syn['molecule_synonym'])
+            if raw_syn['syn_type'] == 'SYSTEMATIC':
+                iupac_name = raw_syn['molecule_synonym']
```
Comment (juanfmx2) [inline src/glados/schema_tags_generator.py]: if the IUPAC name is found there is no need to continue iterating put a break to escape from the for. However, worst case scenario there should be less than 1000 synonyms, so it is not crucial.

**Comment by nclopezo** [comment]
Ghost inspector tests were manually reviewed here:
http://wwwdev.ebi.ac.uk/chembl/
Everything passed

File: src/glados/schema_tags_generator.py (hunk 1 above)
Comment (ricardoaat) [reply src/glados/schema_tags_generator.py]: But the for is used for filling the alternate_names_set, I hijacked it for that conditional. In theory there should be only one alternate name of the type "systematic"

**Comment by nclopezo** [comment]
This is now deployed now:
https://www.ebi.ac.uk/chembl/compound_report_card/CHEMBL266128/

**Pull request merged**